primitives/timestamp/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "sp-timestamp"
+version = "2.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+sr-api = { path = "../sr-api", default-features = false }
+rstd = { package = "sr-std", path = "../sr-std", default-features = false }
```

Comment:
no renames?

Reply:
I mostly kept the existing names and copied `Cargo.toml` files to keep the changes low. I would fix this in the upcoming PR if you don't mind.

Reply:
Sure!